src/smockit/smockit.ts

```diff
@@ -13,7 +13,8 @@ import {
   SmockSpec,
 } from './types'
 import { bindSmock } from './binding'
-import { toHexString, fromHexString, makeRandomAddress } from '../utils'
+import { toHexString, fromHexString } from '@eth-optimism/core-utils'
```

Comment:
General note that we separate out internal/external imports for readability. So this would go above in `Imports: External`. I'll leave a note about this in all of the relevant places in this PR. If you happen to be adding new imports to a file that doesn't have an `Imports: External` section already, plz make sure to create that comment/header thing.

Reply:
i feel ashamed for completely missing that, sorry!

Reply:
I generally do the same in my own file. 
I think I missed them because I was trying to rush to see your twitch stream 😅